eggdrop-basic.conf

```diff
@@ -162,13 +162,14 @@ set prefer-ipv6 0
 
 ## See eggdrop.conf for documentation on the logfile flags and settings.
 
-## This creates a logfile named eggdrop.log containing private msgs/ctcps,
-## commands, errors, and misc. info from any channel.
+# This creates a logfile named eggdrop.log that captures items logged at the
+# m, c, and o flag levels (private msgs/ctcps, commands/errors, and misc 
```

Comment:
You changed `log` by `flag` instead of `level` by `flag`. Same for all below and same sed as above.

Reply:
Done